core/src/main/scala/org/http4s/multipart/MultipartParser.scala

```diff
@@ -19,7 +19,10 @@ object MultipartParser {
 
   private final case class Out[+A](a: A, tail: Option[ByteVector] = None)
 
-  def parse(boundary: Boundary): Writer1[Headers, ByteVector, ByteVector] = {
+  def parse(boundary: Boundary): Writer1[Headers, ByteVector, ByteVector] =
+    parse(boundary, 10 * 1024)
+
+  def parse(boundary: Boundary, headerLimit: Long): Writer1[Headers, ByteVector, ByteVector] = {
```

Comment:
If we are going to switch to a default, couldn't we give the second call the default value, and then call the second call with the first, and then phase out the first in 0.16?